cpp/ql/lib/semmle/code/cpp/ir/implementation/raw/internal/TranslatedStmt.qll

```diff
@@ -596,18 +667,28 @@ class TranslatedBlock extends TranslatedStmt {
     else result = this.getStmt(0).getFirstInstruction(kind)
   }
 
+  override Instruction getALastInstructionInternal() {
+    if this.isEmpty()
+    then result = this.getInstruction(OnlyInstructionTag())
+    else result = this.getStmt(this.getStmtCount() - 1).getFirstInstruction(any(GotoEdge goto))
+  }
+
+  override TranslatedElement getLastChild() {
+    not this.isEmpty() and result = this.getStmt(this.getStmtCount() - 1)
```

Comment:
Isn't `not this.isEmpty()` implied by `result = this.getStmt(this.getStmtCount() - 1)`?

Reply:
Oh, true.